_guide_topics/rhinocommon/compute/compute-getting-started.md

```diff
@@ -0,0 +1,306 @@
+- [NewtonSoft.JSON](https://www.nuget.org/packages/Newtonsoft.Json/) - The very popular JSON library.  Compute communicates using a JSON format in the body of a REST POST. This is available as a Nuget Package. This package is called directly by RHinoCompute.cs.
+- [RhinoCompute.cs](https://compute.rhino3d.com/sdk/csharp) - This is a work in progress package which is meant to add classes available in RhinoCommon, but not available through Rhino3dmIO. RhinoCompute makes calls into RhinoCompute for these functions.
+
+Here are step by step instructions to setting up a basic project to use Compute:
+
+#### File New
+
+1. If you have not done so already, *launch Visual Studio* (for the purposes of this guide, we are using Visual Studio 2017 Community Edition and C#).
+1. Navigate to *File* > *New* > *Project*...
+![File New Project]({{ site.baseurl }}/images/your-first-plugin-windows-01.png)
+1. A *New Project* wizard should appear.  In the left column, find the *Installed* > *Visual C#* section.  In the central list, select the *Console App (.NET Framework)* template...
+![New Project]({{ site.baseurl }}/images/compute-start.png)
+1. For the purposes of this Guide, we will name our demo plugin *TestCompute*.  At the bottom of the window, fill in the *Name* field.  *Browse* and select a location for this plugin on your disk...
+1. Check *Create directory for solution*.  *Note*: This is optional depending on how you want to structure your projects.
+1. Click the *OK* button.  *Note*: You don't have to add the project to source control for this demo.
+1. For the purposes of this guide, we will *accept the defaults* and click *Finish*...
+1. A *new solution* called *HelloRhinoCommon* should open...
+
+#### Steps to install the NuGet packages
+
+1. *Right-click* your project file in *Solution Explorer* and select *Manage NuGet Packages ...*.
+1. On the left side of the dialog expand the *Online* option and select *nuget.org*.
+1. In the top right search box type "Rhino3dmIO" and click on a *Rhino3dmIO.dll* option (there are 3: *x86*, *x64*, and *AnyCPU*) and click on the *Install* button.
+1. Continue by typing in the search box type "Newtonsoft.JSON" and click on a *NewtonSoft.JSON* option and click on the *Install* button.
+1. Close the *Manage NuGet Packages* dialog.  The Nuget packages are installed and ready to use.
+
+Changes that were made:
+
+- The *Rhino3dmIO and NewtonSoft.JSON NuGet packages* are installed in your project.
+- The project references the *Rhino3dmIO* and *Newtonsoft.JSON* assembly.
+- The project's *Post-build event* has been modified so the *rhino3dmio_native.dll* and newtonsoftljson.dll* gets copied to the same output directory as *TestCompute.exe* when the project is built.
```

Comment:
The post-build event, added by Rhino3dmIO.Desktop, only deals with the native Rhino3dmIO libs. Newtonsoft.Json.dll gets copied to the output directory by default because it's referenced by RhinoCompute.cs. I don't think it's worth mentioning this.